lib/routes/home/widgets/payments_list/dialog/closed_channel_payment_details.dart

```diff
@@ -18,11 +18,12 @@ class ClosedChannelPaymentDetailsWidget extends StatelessWidget {
   Widget build(BuildContext context) {
     final themeData = Theme.of(context);
     final texts = context.texts();
-    return FutureBuilder<Config>(
-      future: Config.instance(),
-      builder: (BuildContext context, AsyncSnapshot<Config> snapshot) {
+    final mempoolHelper = MempoolHelper();
+    return FutureBuilder<String>(
+      future: mempoolHelper.blockexplorer,
+      builder: (BuildContext context, AsyncSnapshot<String> snapshot) {
         if (snapshot.hasData) {
-          final blockExplorer = snapshot.data!.defaultMempoolUrl;
```

Comment:
why are you renaming `blockExplorer` to a single word `blockexplorer` but you are creating a `mempoolExplorer` at https://github.com/breez/c-breez/pull/684/files#diff-79e700980e37c83684860b5e1ab0c9fd408829962a7ad97005f1930010ea25f7R53

Reply:
Good point, fixed and renamed to mempoolInstance